README.md

```diff
@@ -106,6 +108,15 @@ Consumers who are trusting images based on the ref-engine discovery and ref-engi
 
 Consumers who are trusting images based on a property of the Merkle tree (e.g. [like this][signed-name-assertions]) can safely perform ref-engine discovery and ref-resolution over HTTP, although they may still want to use `--protocol=https` to protect from sniffers.
 
+This project can also build out the binary tool by Go implementation and exec the command line:
+
+```
+$ make oci-discovery
+$ ./oci-discovery --debug resolve example.com/app#1.0
+```
+
+It presents the same stdout as Python interface when discovering the same server.
```

Comment:
That's the goal, but it's not where we are yet.  I'd rather be linking to `tools/README.md` for details on this.